packages/mui-material/src/Accordion/accordionClasses.ts

```diff
@@ -1,4 +1,4 @@
-import { unstable_generateUtilityClasses as generateUtilityClasses } from '@mui/utils';
+import generateUtilityClasses from '@mui/utils/generateUtilityClasses';
```

Comment:
With `generateUtilityClasses`, we could import the local Material UI version (similarly to generateUtilityClass). It would allow us to customize the class name pattern.

```suggestion
import generateUtilityClasses from '../generateUtilityClasses';
```